src/wallet/sqlite.h

```diff
@@ -36,11 +36,20 @@ class SQLiteCursor : public DatabaseCursor
     Status Next(DataStream& key, DataStream& value) override;
 };
 
+class SQliteExecHandler
```

Comment:
Yeah, this was meant to be a class encapsulating all sqlite functions so that we can trigger different errors and improve and test our error-handling code. I'm not sure why I implemented `CanExecute` instead of decoupling the `Exec()`, but, well, I just changed the implementation to follow the initial idea. Thanks for pointing this out.